packages/contracts-bedrock/book/src/policies/release-process.md

```diff
@@ -4,6 +4,7 @@
 
 First select a tag string based on the guidance in [Monorepo Contracts Release Versioning](./versioning.md#monorepo-contracts-release-versioning)
 
+1. Before creating a finalized release (i.e. not a release candidate), you MUST have the contracts deployed on Sepolia and Mainnet and work with the EVM Safety team to perform the [Contracts Release Checklist](https://www.notion.so/oplabs/Contracts-Release-Checklists-216f153ee16280fda3c2f141c062f974?source=copy_link)
```

Comment:
I think a neater solution here would be to link to the "finalizing a release" section below, and add the context about the release checklist there. The advantage is that this section already states that finalizing should happen after governance has passed (another thing we missed with U17).